src/org/robockets/steamworks/subsystems/Conveyor.java

```diff
@@ -0,0 +1,23 @@
+package org.robockets.steamworks.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.robockets.commons.YAxisRelativeDirection;
+
+/**
+ * @author Jake Backer
+ */
+public class Conveyor extends Subsystem {
+
+    public void initDefaultCommand() {
+
+    }
+
+    /**
+     * Move the conveyor belt in the specified direction
+     * @param direction The direction in which to move the conveyor
+     */
+    public void moveConveyor(YAxisRelativeDirection direction) {
```

Comment:
A change has been pushed to Robockets-Commons that gets rid of `YAxisRelativeDirection`. This is fine for now, but it will have to be changed soon.